substra/cli/interface.py

```diff
@@ -74,14 +74,27 @@ def display(res):
     print(res)
 
 
-# TODO profile, config, json and verbose options should be handled in a single
-#      decorator to populate a GlobalOption object stored in the context
+class GlobalConf:
+    profile = None
+    config = None
+    user = None
+    verbose = None
+    expand = None
+    output_format = None
+
+
+def save_to_global_conf(ctx, param, value):
+    if not hasattr(ctx, 'global_conf'):
+        setattr(ctx, 'global_conf', GlobalConf())
+    setattr(ctx.global_conf, param.name, value)
 
 
 def click_option_profile(f):
```

Comment:
I think we should renamed these functions. What do you think about `click_global_conf_profile`?
It will be easier for developers when reading the code to understand that these options are available through the `ctx.GlobalConf` instance and not as input variable.

Reply:
Yes, that's a better name I think, I'll rename the function :+1: